gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/filter/Range.feature

```diff
@@ -191,7 +191,7 @@ Feature: Step - range()
       """
       g.V().fold().range(Scope.local, 6, 7)
       """
-    When iterated to list
+    When iterated next
```

Comment:
Result of empty list can be represented as:
```
    When iterated to list
    Then the result should be unordered
      | result |
      | l[] |
```
I'd suggest either change this one or at least add another test for the empty list return.